hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordPayload.java

```diff
@@ -140,4 +141,15 @@ default Comparable<?> getOrderingValue() {
     // default natural order
     return 0;
   }
+
+  static String getAvroPayloadForMergeMode(RecordMergeMode mergeMode) {
+    switch (mergeMode) {
+      //TODO: After we have merge mode working for writing, we should have a dummy payload that will throw exception when used
+      default:
```

Comment:
Should `CUSTOM` mode be handled too?  Is it easier to store the payload class name in the `RecordMergeMode` enum itself?

Reply:
CUSTOM doesn't have an associated payload class. For now we will default to the default payload. But once we fix the experience of using engine native records in the writer, we should not be using payloads anymore.

Reply:
Sg.